content/en/docs/Installing and Configuring DreamFactory/_index.md

```diff
@@ -211,6 +211,18 @@ Immediately following this prompt you'll be informed of successful setup:
 	* your instance using a browser. You can run "php artisan serve"
 	* to try out your instance without setting up a web server.
 	**********************************************************
+After creating an Admin, your instance may show the following error:
+```
+Admins
+There is no root administrator selected. Some functionality might not work. Use df:root_admin command to choose one.
+```
+
+A very simple approach to solve it, is to reset the root from inside your container:
+`$ php artisan df:root`
```

Comment:
This should be df:root_admin
df:root works, but given the error message tells you to use df:root_admin, and also if you run 'php artisan' inside df, it gives you the df:root_admin option but not df:root

This would avoid confusion